srcpkgs/dev86/template

```diff
@@ -1,31 +1,37 @@
 # Template file for 'dev86'
 pkgname=dev86
 version=0.16.21
-revision=4
-archs="i686* x86_64*"
```

Comment:
Is this because you build functional dev86 on ppc/arm?

Reply:
Yeah, it appears to work when natively built on aarch64.